2_process/src/make_lake_metadata_augmented_model_prep.py

```diff
@@ -0,0 +1,42 @@
+import os
+import pandas as pd
+
+def add_area_from_file(in_file, area_file, out_file, area_column_in, area_column_out):
+    """
+    Add column with lake surface areas to metadata and save to new file.
+    
+    Code smell: This function is similar to
```

Comment:
😲 I've never heard/seen this term before! It is indeed quite similar, but I think it makes sense to just flag it for now (as you've done) rather than taking the time to write out a more flexible function.

Reply:
Haha, not sure where I picked that term up! Maybe Jordan? Anyway, I agree - the nearly-duplicate code isn't ideal, but it's not worth refactoring at this point.